src/main/java/perosnal/spotifymatcher/service/SpotifyApiService.java

```diff
@@ -86,5 +88,23 @@ public List<Track> getTracksDetails(List<String> tracksIds, String token) {
                 .collect(Collectors.toList());
     }
 
+    private void insertFakeUser(User user) {
+        user.getMatches().stream()
+                .filter(match -> match.equals("fake " + user.getEmail()))
+                .findAny()
+                .ifPresentOrElse(value -> {
+                            user.getMatches().remove(value);
+                            userRepository.save(user);
+                        },
+                        () -> userRepository.save(User.builder()
```

Comment:
Maybe instead of saving in user repository, we could modify the method for getting matches and return a random list of matches? (e.g. with seed based on user's email)

This way you won't be adding random stuff to your actual DB (:

Reply:
Good idea. Check now :)